### ISSUE TITLE
Hot pink highlight in VS 2015

### ISSUE THREAD

**Issue by styfle** [lyphtec/solarized#1]
When I put my cursor on an identifier, such as a method call, it highlights in hot pink.

![image](https://cloud.githubusercontent.com/assets/229881/9419807/cf397c60-4813-11e5-8ec4-432595e23a05.png)

But when I put my cursor on the identifier for the method definition, it highlights in the expected dark blue.

![image](https://cloud.githubusercontent.com/assets/229881/9419811/e32ebbcc-4813-11e5-8651-e919670f7f03.png)

Is this correct? I would expect both to be the same color, preferably the latter.

**Comment by lyphtec** [comment]
The "hot pink" is part of Solarized as magenta (#d33682).

The setting that currently sets this colour is "Highlighted Reference":

![image](https://cloud.githubusercontent.com/assets/804473/9424126/d3a1b19e-4923-11e5-80cb-5b108393d12b.png)

I've tried to play around with the alpha channel (transparency) values to make it not so bright but it appears that the alpha channel values doesn't seem to do anything.

I originally chose magenta because I wanted to very quickly see where my declared variable was being used:

![image](https://cloud.githubusercontent.com/assets/804473/9424154/c3ffd080-4924-11e5-93a8-c4bc6a421c91.png)

As for why the declared identifier is not also the same colour as all the references - I'm yet to figure out where to change the colour values for this as in VS2015 - this seems to now have it's own setting rather than use the same setting as "Highlighted Reference"

**Comment by styfle** [comment]
I think the #d33682 color you are using would be better suited for CTRL + F because that that point, the user is attempting to find matches. Currently this uses a dark red/pink/magenta color.

Just typing or moving the cursor on the variable doesn't meant the user want's to see all the matches and can be distracting.

Look at this example when writing HTML.

Cursor clicks "div"
![image](https://cloud.githubusercontent.com/assets/229881/9479944/bad19fb8-4b34-11e5-945c-52284b60114d.png)

Cursor click "textarea"
![image](https://cloud.githubusercontent.com/assets/229881/9479972/e478131a-4b34-11e5-9488-6c36bb6b3595.png)

It's a bit distracting for me so I changed to
Background: FF073642
Foreground: FF477786

And then I changed Find Match Highlight to FFD33682

![image](https://cloud.githubusercontent.com/assets/229881/9480182/89fd5e98-4b36-11e5-9b72-925be7db3a78.png)
